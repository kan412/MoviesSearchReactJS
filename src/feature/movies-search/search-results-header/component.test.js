import React from 'react';
import { shallow } from 'enzyme';
import SearchResultsHeader from './component';

describe('<SearchResultsHeader />', () => {
  it('render without breaking', () => {
    const searchRes = 10;
    const component = shallow(<SearchResultsHeader searchResults={searchRes} />);
    expect(component).toMatchSnapshot();
  });
});
