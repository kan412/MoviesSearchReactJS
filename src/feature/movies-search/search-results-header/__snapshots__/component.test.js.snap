// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchResultsHeader /> render without breaking 1`] = `
<div
  className="search-results-header"
>
  <div
    className="search-results-header-inner"
  >
    <div
      className="search-results-found"
    >
      10
      <span>
         movies found
      </span>
    </div>
    <Connect(SortContainer) />
  </div>
</div>
`;
